source/auth/auth.rst

```diff
@@ -878,6 +886,13 @@ Q: Why does SCRAM sometimes SASLprep and sometimes not?
 Version History
 ===============
 
+Version 1.8.0 Changes
+    * Test format changed to improve specificity of behavior assertions.
+
+Version 1.7.2 Changes
```

Comment:
Previous version of the spec is 1.7.1. Was 1.7.2 briefly a version during the course of this PR?

Any reason not to just roll these changes into the list for 1.8.0?